plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryWithDifferentProjectIdConnectorSmokeTest.java

```diff
@@ -0,0 +1,116 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.bigquery;
+
+import com.google.common.collect.ImmutableMap;
+import io.trino.plugin.bigquery.BigQueryQueryRunner.BigQuerySqlExecutor;
+import io.trino.testing.BaseConnectorSmokeTest;
+import io.trino.testing.QueryRunner;
+import io.trino.testing.TestingConnectorBehavior;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.util.Map;
+
+import static io.trino.testing.TestingNames.randomNameSuffix;
+import static java.util.Objects.requireNonNull;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestBigQueryWithDifferentProjectIdConnectorSmokeTest
+        extends BaseConnectorSmokeTest
+{
+    private static final String alternateProjectCatalog = "bigquery";
+    private static final String alternateProjectId = requireNonNull(System.getProperty("testing.alternate-bq-project-id"), "testing.alternate-bq-project-id system property not set");
+    private static final String serviceAccountCatalog = "service_account_bigquery";
+
+    protected BigQuerySqlExecutor bigQuerySqlExecutor;
+
+    @BeforeClass(alwaysRun = true)
+    public void initBigQueryExecutor()
+    {
+        this.bigQuerySqlExecutor = new BigQuerySqlExecutor();
+    }
+
+    @Override
+    protected QueryRunner createQueryRunner()
+            throws Exception
+    {
+        QueryRunner queryRunner = BigQueryQueryRunner.createQueryRunner(
+                ImmutableMap.of(),
+                ImmutableMap.of("bigquery.project-id", alternateProjectId),
+                REQUIRED_TPCH_TABLES);
+        queryRunner.createCatalog(serviceAccountCatalog, "bigquery", Map.of(
+                "bigquery.views-enabled", "true",
+                "bigquery.view-expire-duration", "30m"));
+        return queryRunner;
+    }
+
+    @SuppressWarnings("DuplicateBranchesInSwitch")
+    @Override
+    protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
+    {
+        switch (connectorBehavior) {
+            case SUPPORTS_RENAME_SCHEMA:
+                return false;
+
+            case SUPPORTS_RENAME_TABLE:
+                return false;
+
+            default:
+                return super.hasBehavior(connectorBehavior);
+        }
+    }
+
+    @Test
+    public void testCreateSchemasInDifferentProjectIdCatalog()
+    {
+        // This test case would fail without the bug fix
+        // It would create a schema in the wrong project, not the one defined in the catalog properties
+
+        try {
+            assertThat(getQueryRunner().execute("SHOW CATALOGS").getOnlyColumnAsSet())
+                    .contains(alternateProjectCatalog, serviceAccountCatalog);
+
+            getQueryRunner().execute("CREATE SCHEMA " + serviceAccountCatalog + ".service_account_schema");
```

Comment:
Avoid using the static schema name `service_account_schema` & `project_id_schema`. This will lead to flaky tests. 

Also, I would recommend using `assertUpdate()` or other helper methods which is shorter than `getQueryRunner().execute()` here and in other places.